### DISCUSSION TITLE
Bug: The application is unusable

### DISCUSSION THREAD

**Discussion by wiliamjk** [brackets-cont/brackets#333] [Q&A] [answered]
I use Brackets on a MacBook M1 Pro and a few days ago I went to open a folder and accidentally double-clicked on Macintosh HD > Users (instead of clicking once).

After that, Brackets tries to open this folder (which is huge) every time I open the app and, consequently, it freezes. I can't open another folder or another file. I've tried uninstalling the app, but reinstalling it keeps the settings.

What do I do?

**Comment by charlypa** [comment, marked as answer]
@wiliamjk Try deleting the contents in `~/Library/Application Support/Brackets/`. This should reset your Brackets configuration and resolve the freezing issue.

I'd strongly recommend installing Phoenix Code (https://phcode.io) instead of Brackets - it's the next generation of Brackets and offers native binaries specifically optimized for M1 Macs. You'll get much better performance and stability compared to the original Brackets.

**Comment by wiliamjk** [comment]
Thank you very much!
